src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -20,6 +20,9 @@ namespace System.Text
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct Rune : IComparable<Rune>, IEquatable<Rune>
     {
+        internal const int MaxUtf16CharsPerRune = 2; // supplementary plane code points are encoded as 2 UTF-16 code units
+        internal const int MaxUtf8BytesPerRune = 4; // supplementary plane code points are encoded as 4 UTF-8 code units
```

Comment:
should this be used somewhere?

Reply:
Whoops. This was from an early iteration I forgot to revert. Will nuke it.